web/geo/utilities.js

```diff
@@ -0,0 +1,64 @@
+const { polygon, getCoords, getType, featureEach, featureCollection, area, truncate } = require("@turf/turf")
+const fs = require("fs")
+
+function getPolygons(data) {
+    /* Transform the feature collection of polygons and multi-polygons into a feature collection of polygons only */
+    /* all helper functions should rely on its output */
+     const handlePolygon = (feature, props) => polygon(getCoords(feature), props)
+     const handleMultiPolygon = (feature, props) => getCoords(feature).map(coord => polygon(coord, props));
+ 
+     const polygons = [];
+ 
+     const dataGeojsonType = getType(data);
+     if (dataGeojsonType !== "FeatureCollection") {
+         // eslint-disable-next-line no-param-reassign
```

Comment:
It's usually a bad sign if you have to override eslint rules - they are there for a reason 😉

Reply:
@Kongkille I see you marked this as resolved, did you push new code fixing this?

Reply:
Uh no, but I have it on my end. I decided to do a bit of a larger revamp cleaning up the functions, and it was just to keep track of these things. I expect to push it later today.